Source/Shaders/Builtin/Functions/clipPlanes.glsl

```diff
@@ -0,0 +1,38 @@
+/**
+ * Clip a fragment by an array of clipping planes.
+ *
+ * @name czm_clipPlanes
+ * @glslFunction
+ *
+ * @param {vec4[]} clippingPlanes The array of planes used to clip, defined in in eyespace.
+ * @param {int} clippingPlanesLength The number of planes in the array of clipping planes.
+ */
+ const int czm_maxClippingPlanes = 6;
```

Comment:
Remove single space in front of the `const`.

Reply:
The 6 will have to be defined somewhere in the JS side as well. Maybe this should be a property of `UniformState` and `AutomaticUniforms`.

Reply:
I checked the OpenGL documentation and it is fine to pass in more values than the uniform array holds, the extra are ignored. Ignore my other comments about limiting to 6. However it's still worth testing with more than 6 ways to confirm this.

Reply:
Yes, passing in more than 6 just causes the additional planes to be discarded.